.changelog/4375.yml

```
@@ -0,0 +1,4 @@
+changes:
+- description: "Added the option to specify a marketplace to the following playbook fields: inputs, scriptarguments, message, form, left, right, tasks."
```

```suggestion
- description: |-
  Added the option to specify a marketplace for certain playbook fields:
  * All input fields - *key*, *value*, *description*, *required*, *playbookInputKey*).
  * The following task fields: *scriptId*, *scriptarguments*, *loop.scriptArguments*, *message*, *form*.
  * The *left* and *right* options of *condition*/*builtincondition* task fields (and their nested fields).
```
This is very specific and should be documented somewhere for future reference.